api/proto/external.proto

```diff
@@ -416,3 +430,34 @@ message ValidatedMintConfigTx {
     MintConfigTx mint_config_tx = 1;
     Ed25519SignerSet signer_set = 2;
 }
+
+// The amount and blinding factor of a TxOut
+message UnmaskedAmount {
+    // The value of the amount commitment
+    fixed64 value = 1;
+
+    // The token_id of the amount commitment
+    fixed64 token_id = 2;
+
+    // The blinding factor of the amount commitment
+    CurveScalar blinding = 3;
```

Comment:
Do we even need a blinding factor here? If these are going to be unblinded amounts anyway, could we not just have a convention that for unmasked amounts we intentionally choose a blinding factor of 0?

Reply:
Yeah, we do need a blinding factor here, because, the recipient of the `TxOut` may not know that it is unblinded -- they need to be able to recover their balance just from the blockchain. Currently, the blinding factor is derived as a hash of the TxOut shared secret, which is a random value.

We could try to adopt a convention that sometimes the blinding factor is zero, and so view key scanning would have to be modified so that it always tries both the derived value and zero. But this seems bad because it would make view key scanning twice as slow. It seems simpler to just keep using the blinding factor as it is working today.